server/webpack.config.js

```diff
@@ -1,13 +1,31 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
 module.exports = {
   mode: 'production',
   devtool: 'source-map',
   stats: 'errors-only',
   module: {
     rules: [
+      {
+        test: /\.scss$/,
+        use: [
```

Comment:
Loaders run from the bottom up.  Here is what these loaders are doing:
- sass-loader:  Converts Sass into CSS.
- postcss-loader:  Adds vendor prefixes (like `-webkit-`, `-moz-` or `-ms-`) so that styles are compatible with all browsers, even older ones.  USWDS strongly recommends autoprefixing.  See the highlighted note [here](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/#loading-uswds-in-your-sass-entry-point-2).
- css-loader:  Collects all the CSS which resulted from the Sass compilation above and puts it into a single string.
- MiniCssExtractPlugin:  Puts that CSS string into a file.  See the plugin filename setting on line 48.

Note that all of these loaders are 3rd-party packages which don't come with Webpack.  That's why I added them to our dependencies in` package.json`.

Reply:
Great explanation! Thank you!

Reply:
Please move this github comment into code comments